.github/workflows/test.yml

```diff
@@ -9,6 +9,20 @@ on:
       - '.gitignore'
 
 jobs:
+  bash-test:
```

Comment:
🚀 Consider adding a description for the 'bash-test' job to clarify its purpose in the workflow.